srcpkgs/SwayOSD/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'SwayOSD'
+pkgname=SwayOSD
+version=0.2.1
+revision=1
+build_style=meson
+build_helper="rust"
+hostmakedepends="pkg-config ninja sassc cargo"
+makedepends="gdk-pixbuf-devel glib-devel cairo-devel pango-devel graphene-devel gtk4-devel gtk4-layer-shell-devel libevdev-devel libinput-devel eudev-libudev-devel"
```

Comment:
It's better to put `glib-devel` in hostmakedepends, since `glib-compile-resources` runs on the host.

```suggestion
hostmakedepends="pkg-config ninja sassc cargo glib-devel"
makedepends="gdk-pixbuf-devel cairo-devel pango-devel graphene-devel gtk4-devel gtk4-layer-shell-devel libevdev-devel libinput-devel eudev-libudev-devel"
```